Makefile

```diff
@@ -7,6 +7,7 @@ OPERATOR_IMAGE ?= node-maintenance-operator
 REGISTRY_IMAGE ?= node-maintenance-operator-registry
 
 TARGETS = \
+	cluster-up \
```

Comment:
What about creating a target for refreshing the subtree?

Reply:
add pull-ci-changes target to Makefile.